spec/features/11_destroy_review_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "Destroy Movie" do
+  let!(:user1) { FactoryGirl.create(:user) }
+  let!(:user2) { FactoryGirl.create(:user) }
+  let!(:movie1) { FactoryGirl.create(:movie) }
+  let!(:review1) { FactoryGirl.create(:review, movie_id: movie1.id, user_id: user1.id) }
+
+  before do
+    user_sign_in(user1)
+  end
+
+  scenario "User clicks link delete review, and is redirected to"\
+    " movie show page" do
+    visit root_path
```

Comment:
Similar to above comments. These can be moved into the before block. You can also just navigate to the page you want instead of going to the root first